Ignore Style/WordArray in languages_helper

nschonni opened the pull request:
I regenerated the TODO and noticed the detection for the Style/WordArray had changed, so I just moved it to in inline ignore.

nschonni committed:
Ignore Style/WordArray in languages_helper

mjankowski reviewed (approved):
The thing the rule wants you to do here makes sense one-off for arrays, but looks really bizarre in this large collection of arrays within the hash (winds up mixing styles).

Agree that disable makes more sense.

nschonni commented:
@mjankowski I rebased this one and dropped the TODO which kind of has your suggestion. Feel free to submit one that hoists that config to the root though

mjankowski commented:
@nschonni I just want to say that I appreciate your persistence in rebasing this PR.

mjankowski commented:
I love this PR and I will never close it. I'm looking forward to when you next rebase it.

nschonni commented:
Rebased and re-ran the autofix for you @mjankowski 😆

mjankowski commented:
![oh-yeah-randy-savage](https://github.com/user-attachments/assets/9eddf9db-7d2b-45e0-8927-f0edf13ed9a3)

nschonni commented:
no problem, I rebased and re-ran, which found one more

mjankowski commented:
The dream is alive.

mjankowski commented:
Closing as handled by https://github.com/mastodon/mastodon/pull/33580

I'll never forget the good times we had here. The ups and downs. The triumphs and the defeats. What a rollercoaster ride.
